.depcheckrc.json

```diff
@@ -0,0 +1,22 @@
+{
+  "ignores": [
+    "@lavamoat/allow-scripts",
+    "@lavamoat/preinstall-always-fail",
+    "@metamask/auto-changelog",
+    "@types/*",
+    "prettier-plugin-packagejson",
+    "ts-node",
+    "typedoc",
+    "@storybook/addon-actions",
+    "@storybook/react-webpack5",
+    "babel-loader",
+    "dependency-tree",
+    "eslint-plugin-node",
+    "sass",
+    "webpack",
+    "@yarnpkg/core",
+    "@yarnpkg/cli",
+    "clipanion",
+    "@yarnpkg/fslib"
```

Comment:
These packages are listed as missing dependencies when they shouldn't be required and are also not listed in the module template `.depcheckrc.json` file

https://github.com/MetaMask/metamask-module-template/blob/main/.depcheckrc.json

Reply:
Resolved by @mcmire comment here https://github.com/MetaMask/design-tokens/pull/600/files#r1474934441